geode-core/src/integrationTest/java/org/apache/geode/management/internal/cli/util/LogExporterFileIntegrationTest.java

```diff
@@ -148,4 +148,40 @@ public void findLogsWhichContainsTheWordLog() throws Exception {
     assertThat(logExporter.findLogFiles(workingDir.toPath()))
         .contains(gcRolledOverLogFile.toPath());
   }
+
+  // GEODE-6706
+
+  @Test
+  public void findLogsWhichContainsEmptyConfiguredGCDir() throws Exception {
+    System.setProperty("-Xloggc", "");
+
+    File gcLogFile = new File(workingDir.getAbsolutePath(), "gc.log");
+    FileUtils.writeStringToFile(gcLogFile, "some gc log line");
+
+    File gcRolledOverLogFile = new File(workingDir.getAbsolutePath(), "gc.log.1");
+    FileUtils.writeStringToFile(gcRolledOverLogFile, "some gc log line");
+
+    assertThat(logExporter.findLogFiles((new File(workingDir.getAbsolutePath()).toPath()))
+        .contains(gcLogFile.toPath()));
+    assertThat(logExporter.findLogFiles((new File(workingDir.getAbsolutePath()).toPath()))
+        .contains(gcRolledOverLogFile.toPath()));
+  }
+
+  @Test
+  public void findLogsWhichContainsGCLogFromConfiguredGCDir() throws Exception {
+    System.setProperty("-Xloggc", workingDir.getAbsolutePath());
```

Comment:
"-Xloggc" is a magic string used throughout this change, and is a private-static of production code. Can it at minimum be a private static of test code as well so that it is not replicated everywhere?